yaksh/decorators.py

```diff
@@ -1,20 +1,47 @@
-from django.shortcuts import render_to_response
+from django.shortcuts import render_to_response, redirect
 from django.conf import settings
 from django.template import RequestContext
 
+# Local imports
+from yaksh.forms import ProfileForm
+
+
+def user_has_profile(user):
+    return hasattr(user, 'profile')
+
+
+def has_profile(func):
+    """
+    This decorator is used to check if the user account has a profile.
+    If the user does not have a profile then redirect the user to
+    profile edit page.
+    """
+
+    def _wrapped_view(request, *args, **kwargs):
+        if user_has_profile(request.user):
+            return func(request, *args, **kwargs)
+        ci = RequestContext(request)
+        template = 'manage.html' if 'moderator' in request.user.groups.all() else 'user.html'
```

Comment:
This leads to cyclic import causing an ImportError since views also imports functions from decorators and decorator will import from views.

Adding the underlying condition from ```is_moderator``` here until a better solution is found